### PR TITLE
Add date field and transaction link.

### PR DESCRIPTION
**Pull request by glaush**

### CODE REVIEW

File: ui/view/controls/OpenInBlockchainExplorer.qml [hunk 1]
```diff
@@ -12,17 +12,6 @@ Item {
     Layout.preferredWidth: (control.showText ? openInExplorer.width + 10 : 10) + openInExplorerIcon.width
     height: 16
 
-    SFText {
```
Comment (echoby) [inline ui/view/controls/OpenInBlockchainExplorer.qml:15]: why deleted? "showText = false" will disable this

File: ui/view/wallet/TransactionDetailsPopup.qml [hunk 2]
```diff
@@ -142,6 +143,27 @@ CustomDialog {
...
             
+            SFText {
+                font.pixelSize: 14
+                color: Style.content_secondary
+                //% "Date"
+                text: qsTrId("tx-details-date-label") + ":"
+                visible: dateField.parent.visible
+            }
+            RowLayout {
+                visible: stm.state == "tx_info" && dialog.sendAddress.length && !(isIncome && isShieldedTx)
```
Comment (echoby) [inline ui/view/wallet/TransactionDetailsPopup.qml:154]: why you hide this field? The date field available for all transaction types.

File: ui/view/controls/OpenInBlockchainExplorer.qml (hunk 1 above)
Comment (glaush) [reply ui/view/controls/OpenInBlockchainExplorer.qml:15]: Fixed.

File: ui/view/wallet/TransactionDetailsPopup.qml (hunk 2 above)
Comment (glaush) [reply ui/view/wallet/TransactionDetailsPopup.qml]: Oh, you're a hawkeye. It seemed to me from the design that specifically for some transactions.

Comment (glaush) [reply ui/view/wallet/TransactionDetailsPopup.qml]: Fixed.

**Pull request merged**